app/grandchallenge/reader_studies/serializers.py

```diff
@@ -74,8 +76,19 @@ class Meta:
             "overlay_segments",
             "look_up_table",
             "widget",
+            "widget_options",
         )
 
+    def get_widget_options(self, obj: Question) -> dict:
```

Comment:
I think the API is clearer if the values are just serialized directly rather than grouped up into a dictionary, makes it easier for the client to work with.

Reply:
Yes, the client can then check all the ones that are not null, just like we do server side.